src/LinAlg/SparseMatrix.C

```diff
@@ -695,6 +693,17 @@ void SparseMatrix::initAssembly (const SAM& sam, bool delayLocking)
   default: break;
   }
   // The sparsity pattern is now permanently locked (until resize is invoked)
+}
+
+
+void SparseMatrix::initAssembly (const SAM& sam, bool delayLocking)
+{
+  this->resize(sam.neq,sam.neq);
+#ifdef USE_OPENMP
+  if (omp_get_max_threads() < 2)
+    return;
+
+  preAssemble(sam, delayLocking);
```

Comment:
Heller slik istedet for 703-706:

if (omp_get_max_threads() > 1)
  this->preAssemble(sam,delayLocking);